### DISCUSSION TITLE
Social media strategy

### DISCUSSION THREAD

**Discussion by RiteshBalayan** [RiteshBalayan/instaorbit#10] [General]

**Comment by RiteshBalayan** [comment]
Should we start with a Linkdin post to let close people know we are doing a remarkable work and Space and Physics.

**Reply by hrithikthakur** [reply]
Actually, twitter and (tiktok/ instagram?) seems ideal. If we intially don't get a likes on LinkedIn or youtube, the algorithm woukd become our enemy and we might build a bad repututaion. we should post on relevant twitter communities and then from this and the blogs on the websites, once we get traction there, we can move to LinkedIn.

**Reply by RiteshBalayan** [reply]
Twitter. ... Let's do it.

**Reply by hrithikthakur** [reply]
also, look at how https://x.com/ayushswrites is marketing his new startup on twitter. we can take a few pointers from him.

**Comment by hrithikthakur** [comment]
### Some Marketing Tactics
- creating a "movement" for cheap space mission management iterations.
- the "make in india" movement. Highlight the more-than-required use of foreign software products

**Reply by RiteshBalayan** [reply]
This one might be really impactful "Highlight the more-than-required use of foreign software products" Let's use this one on our marketing campain of india in every conversation.

**Comment by hrithikthakur** [comment]
### Long-term tactics for Brand Identity:
- getting a blue checkmark badge on Gmail for the company email address, aside from twitter.
